docs/platforms/javascript/common/opentelemetry/sampler.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Using a Custom OpenTelemetry Sampler
```

Comment:
```suggestion
title: Custom OpenTelemetry Sampler
```
It would be great if rather than making this a standalone page we could add it to either the "using a custom OTEL setup" or "using custom OTEL instrumentation" page. Would love to hear your thoughts!